app/assets/stylesheets/chat.scss

```diff
@@ -8,7 +8,7 @@
   width: 100%;
   max-width: calc(1540px + 10vw);
 
-  --layout-chat-left-sidebar-width: 260px;
+  --layout-chat-left-sidebar-width: 250px;
```

Comment:
Would you mind explaining this a little? It's 10px diff so I wonder what's exactly going on there :)

Reply:
I feel the left sidebar width was too much for the iPad size, so I reduce 10 px.

Reply:
Hey @Veggier, can we keep it as it was on 260px? We have some tabs and icons there that would wrap to another line if we reduce it to 250px.

| 260px | 250px |
|---|---|
| ![image](https://user-images.githubusercontent.com/108287/103339770-915e1c80-4a82-11eb-94e8-c2770388b502.png) | ![image](https://user-images.githubusercontent.com/108287/103339798-af2b8180-4a82-11eb-8307-3c3c8aef0aaf.png) |

Especially it's not a big difference visually so I don't think it's really necessary.

Reply:
@ludwiczakpawel I changed it back now.